### PR TITLE
Remove remaining advocacy.code.org references from the codebase

### PR DESCRIPTION
**Pull request by kelbyhawn**
Removes the remaining references to advocacy.code.org in the codebase.

## Links
Jira ticket: [ACQ-2086](https://codedotorg.atlassian.net/browse/ACQ-2086)
Related PRs: 
- https://github.com/code-dot-org/code-dot-org/pull/59844
- https://github.com/code-dot-org/code-dot-org/pull/59892

## Testing story
Made sure drone passes

### CODE REVIEW

**Comment by kelbyhawn** [comment]
This is the only place I'm not sure about, does anyone have an idea on how to update this or if we should hand it off to Platform? https://github.com/code-dot-org/code-dot-org/blob/e8047094c639e57fffc131f7f1b6cd7990d70d18/lib/cdo/cloud_formation/cdo_app.rb#L190

**Comment by cat5inthecradle** [comment]
> This is the only place I'm not sure about, does anyone have an idea on how to update this or if we should hand it off to Platform?
> 
> https://github.com/code-dot-org/code-dot-org/blob/e8047094c639e57fffc131f7f1b6cd7990d70d18/lib/cdo/cloud_formation/cdo_app.rb#L190

@code-dot-org/infrastructure can look into what exactly this is doing.

**Comment by cat5inthecradle** [comment]
@kelbyhawn I think you can just remove the appending of 'advocacy' to the array.

image

**Comment by kelbyhawn** [comment]
> I think you can just remove the appending of 'advocacy' to the array.

@cat5inthecradle thank you, just updated!

File: lib/cdo/cloud_formation/cdo_app.rb [hunk 1]
```diff
@@ -187,7 +187,7 @@ def cloudfront_config(app)
         subdomain('origin'),
         app == 'Dashboard' ?
           [studio_subdomain] :
-          [subdomain] + (CDO.partners + ['advocacy']).map {|x| subdomain(nil, x)},
+          [subdomain] + (CDO.partners).map {|x| subdomain(nil, x)},
```
Comment (bethanyaconnor) [inline lib/cdo/cloud_formation/cdo_app.rb]: nit: remove parentheses

**Comment by kelbyhawn** [comment]
@bethanyaconnor @cat5inthecradle this is passing drone now 🎉 I removed all of the CSEdWeek cleanup and made a new ticket for that [here](https://codedotorg.atlassian.net/browse/ACQ-2127).

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**